README.md

```diff
@@ -76,6 +79,9 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get('/', function (req, res){
```

Comment:
We needed to create one route for each of the main HTTP request methods and `send` a string representing the method name:

```js
app.get('/', (req,res)=>(
    res.send('GET')
))

app.post('/', (req,res)=>(
    res.send('POST')
))

app.put('/', (req,res)=>(
    res.send('PUT')
))

app.delete('/', (req,res)=>(
    res.send('DELETE')
))
```